docs/library/axi_ad9144/index.rst

```diff
@@ -122,6 +122,35 @@ system's AXI bus, facilitating efficient data transfer and control.
 Register Map
 --------------------------------------------------------------------------------
 
+The register map of the core contains instances of several generic register maps
+like DAC common, DAC channel, JESD TPL.
+The following table presents the base addresses of each instance, after it you
+can find the detailed description of each generic register map.
+
+The absolute address of a register should be calculated by adding the instance
+base address to the registers relative address. For a more detailed explanation,
+see :ref:`DAC register access <generic-dac-register-access>`.
+
+.. list-table:: Register Map base addresses for ad_ip_jesd204_tpl_dac
+   :header-rows: 1
+
+   * - HDL reg
+     - Software reg
+     - Name
+     - Description
+   * - 0x0000
+     - 0x0000
+     - BASE
+     - See the `Base <#hdl-regmap-COMMON>`__ table for more details.
+   * - 0x0000
```

Comment:
For this IP the default offset was probably 0x1000 => 0x4000 based on https://github.com/analogdevicesinc/hdl/blob/main/library/common/up_dac_common.v#L49

But I would remove the documentation for axi_ad9144, since this ip was replaced with the generic dac_jesd_tpl a long time ago.
The dac_jesd_tpl has this set to 0x0 https://github.com/analogdevicesinc/hdl/blob/main/library/jesd204/ad_ip_jesd204_tpl_dac/ad_ip_jesd204_tpl_dac_regmap.v#L214
Older dac ips have this generic 0x4000 offset.